src/main/java/cs2103/v15_1j/jimjim/command/ShiftCommand.java

```diff
@@ -77,7 +78,7 @@ public UIFeedback execute(ControllerStates conStates) {
         	actual = conStates.displayList.getTaskEvent(taskNum-1, prefix);
             Event tempEvent = null;
         	if (actual instanceof FloatingTask || actual instanceof DeadlineTask) {
-        		return new FailureFeedback("Can't shift a task!");	
+        		return new InvalidFeedback("Can't shift a task!");	
```

Comment:
It was an issue in the tracker. Also i think this way is clearer, shows exactly why it fails (through the type of feedback)